src/swagger/api.yaml

```diff
@@ -7140,6 +7160,7 @@ definitions:
         BOOKMARK: '#/definitions/BookmarkApplication'
         BROWSER_PLUGIN: '#/definitions/BrowserPluginApplication'
         OPENID_CONNECT: '#/definitions/OpenIdConnectApplication'
+        SAML_1_1: '#/definitions/SamlApplicationV1'
```

Comment:
For the record, we are trying to achieve https://github.com/okta/openapi/pull/272/files here. 

Mapping two different keys (`SAML_1_1` & `SAML_2_0`) to the same value (`SamlApplication`) is causing our swagger codegen plugin code to break.